src/main/markdown/gettingstarted-v2.md

```diff
@@ -0,0 +1,210 @@
+Getting Started
+===
+
+Starting with 2.11.0 of the GWT SDK the [webAppCreator](gettingstarted-outdated.html) is deprecated. Instead, we will use 
+the [gwt-maven-archetypes](https://github.com/tbroyer/gwt-maven-archetypes) by Thomas Broyer to create a new GWT project. Using the **artifactId: modular-webapp** 
+the plugin generates a Maven project with separate modules for
+
+* client
+* server
+* shared.
+
+**Separation of code is the way to go in modern web development.** 
+
+## TOC
+
+*  [Prerequisites](#prereqs)
+*  [Create your first web application](#create)
+*  [Run locally in super dev mode](#run)
+*  [Make a few changes](#change)
+*  [Debugging the browser](#bebug)
+*  [Compile and run in production mode](#compile)
+*  [Set up an IDE](#setup)
+
+## Prerequisites<a id="prereqs"></a>
+
+1.  GWT 2.11.0 still supports Java 8 (running and building). We suggest to use at least Java 11 or 17. If necessary, download and
+    install the Java SE Development Kit (JDK) <a href="https://adoptium.net/de/temurin/releases/" 
+    rel="nofollow">from Eclipse Temurin</a> for your platform. Install the JDK by following the 
+    installation instructions. 
+2.  Apache Maven is also necessary to run the gwt-maven-archetype. In this sample the Maven CLI is used. If
+    you don't already have it, install <a href="https://maven.apache.org/download.cgi" rel="nofollow">Apache Maven</a>.
+
+If you have problems running the gwt-maven-archetype from the command line, try setting the
+$JDK_HOME environment variable with export JDK_HOME="[pathToJDK]" and $MAVEN_HOME environment variable with export MAVEN_HOME="[pathToMaven]"
+and add both to the path. 
+
+## Create your first web application<a id="create"></a>
+
+First, open a terminal and choose the directory you like to generate the sources into.
+Now, you generate a new application project by using the following command: 
+
+```
+mvn archetype:generate \
```

Comment:
maybe adopt the note about forcing a new version of this plugin, so you don't get extra newlines?

Reply:
I am missing my changes ..